### DISCUSSION TITLE
Android 14在sonic平台的支持

### DISCUSSION THREAD

**Discussion by sunwensheng** [SonicCloudOrg/sonic-server#478] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/bee62b5f-e83d-4f1c-b5f7-fcea81ff529e)
品牌：小米14手机
系统：14
问题描述：
在Sonic打开的手机页面。可以使用底部的菜单按钮回退到首页。但是手机屏幕内无法进行点击和滑动操作。怀疑是Android的UIAutomator没有传递操作的数据。因为在一些安卓11的老系统，这个都是可以的。请问现在Sonic对android14的适配进度。

**Comment by ZhouYixun** [comment, marked as answer]
你的sonic版本是多少呢，最新的是可以的
